.travis.yml

```diff
@@ -61,7 +61,7 @@ jobs:
               - python --version
               - python -m pip install --upgrade pip
               - python -m pip install --progress-bar off poetry
-              - python -m poetry --no-interaction install
-              - python -m poetry --no-interaction build
+              - python -m poetry install --no-interaction 
```

Comment:
Why? What is the purpose of these changes?

Reply:
See https://github.com/datadvance/DjangoChannelsGraphqlWs/pull/37/commits/288f9e75e34a6d3f38d9eda10ee8b39b80125741#r403520450